test/test_multiphysics.py

```diff
@@ -40,6 +62,17 @@
 logger = logging.getLogger(__name__)
 
 
+def get_box_mesh(dim, a, b, n):
```

Comment:
We really need a unified box mesh generator for mirgecom. This appears re-implemented in many places.

Reply:
Yeah, agreed. https://github.com/illinois-ceesd/mirgecom/issues/887